sql_assesment.txt

```diff
@@ -0,0 +1,1786 @@
+
+1) List all the columns of the Salespeople table. 
+
+desc salespeople;
+
++-------+--------------+------+-----+---------+-------+
+| Field | Type         | Null | Key | Default | Extra |
++-------+--------------+------+-----+---------+-------+
+| SNUM  | int          | NO   | PRI | NULL    |       |
+| SNAME | varchar(100) | YES  |     | NULL    |       |
+| CITY  | varchar(100) | YES  |     | NULL    |       |
+| COMM  | int          | YES  |     | NULL    |       |
++-------+--------------+------+-----+---------+-------+
+
```

Comment:
There is a command which would list only column names. You can add that also here.